rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1973,8 +1973,8 @@ def test_upgrade_db_40_to_41(user_data_dir):
         msg_aggregator=msg_aggregator,
         resume_from_backup=False,
     )
-    # add some settings that should move in the upgrade
-    should_move = {
+    # add some settings and used_query_ranges that should/shouldn't move in the upgrade
+    should_move_settings = {
```

Comment:
can you verify in the test that before upgrading you had the expected values in the database at the settings table?